sphinx_ansible_theme/extranav.html

```diff
@@ -4,7 +4,7 @@
 <div id="sideBanner">
   <br/>
   <a href="https://www.ansible.com/docs-left?utm_source=docs">
-    <img style="border-width:0px;" src="https://cdn2.hubspot.net/hubfs/{{ theme_hubspot_id }}/docs-graphics/ASB-docs-left-rail.png" />
+    <img alt="" style="border-width:0px;" src="https://cdn2.hubspot.net/hubfs/{{ theme_hubspot_id }}/docs-graphics/ASB-docs-left-rail.png" />
```

Comment:
Alt shouldn't be empty, then. It hurts accessibility otherwise.

Reply:
Ah, I was assuming this is a tracking pixel. For that, `alt=""` would be exactly the right content.